### PR TITLE
Updated log levels in pkg

### PR DESCRIPTION
**Pull request by akremsa**
This PR can be considered as a part of #1223 

Updated log levels in pkg package.

### CODE REVIEW

File: pkg/sdkserver/localsdk.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func NewLocalSDKServer(filePath string) (*LocalSDKServer, error) {
 
 	go func() {
 		for value := range l.update {
-			logrus.Info("gameserver update received")
+			logrus.Debug("gameserver update received")
```
Comment (aLekSer) [inline pkg/sdkserver/localsdk.go]: Nit: Not connected to the logLevel, but for consistency with log messages above.
```suggestion
			logrus.Debug("GameServer update received")
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -101,3 +101,9 @@ repository, we have a [community membership guide](./docs/governance/community_m
 
 - [How to write a good Git Commit message](https://chris.beams.io/posts/git-commit/) -
   Great way to make sure your Pull Requests get accepted.
+- **Log levels usage:**
```
Comment (markmandel) [inline CONTRIBUTING.md]: Please move this entire section it's own PR. Where possible, smaller PRs are better, especially PRs that touch on multiple aspects.

Here we have a PR that is about log levels in SDK Server, but also touches on contributing guidelines. It's very hard to approve / reject both.

**Comment by akremsa** [comment]
@markmandel  thanks for the review. I've removed changes from the md file and added other log levels updates to this PR because all of them are related to the same package - pkg, and I think it's better to keep everything in a single PR.

**Comment by aLekSer** [comment]
@markmandel I think we could merge this PR before the release. It would significantly reduce logging in SDK server with Info level used by default.
